duckduckgo/iOS
# Small UI Fixes for subscriptions

Task/Issue URL: https://app.asana.com/0/1204099484721401/1207003662391345/f
Tech Design URL:
CC:

**Description**:
A few minor UI changes.

1. Added a title to the VPN navigation bar
2. Added a loader to Restore View

**Steps to test this PR**:
1.  Go to VPN, observe it has a title
2. Go to Subscription > Add to Another Device, observe existing email is loaded or you can add one.

**Here are some images**

**Add to device, slow connection:**
![RocketSim_Recording_iPhone_(17 2)_6 1_2024-04-05_16 43 02](https://github.com/duckduckgo/iOS/assets/1156669/4f73f152-c43e-453a-b314-2537f6e03639)

**VPN Title**
<img width="527" alt="Screenshot 2024-04-05 at 4 49 35 PM" src="https://github.com/duckduckgo/iOS/assets/1156669/d316bc96-f7b4-4696-8e63-c4892ad42084">


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
